examples/different-payload-types/different_payload_types.bal

```diff
@@ -47,23 +47,38 @@ service /actionService on new http:Listener(9090) {
         response = clientEP->post("/echo", binaryValue);
         handleResponse(response);
 
-         //[Create a JSON body part](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/mime/latest/mime/classes/Entity#setJson).
-        mime:Entity part1 = new;
-        part1.setJson({"name": "Jane"});
-
-        //[Create a text body part](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/mime/latest/mime/classes/Entity#setText).
-        mime:Entity part2 = new;
-        part2.setText("Hello");
-
-        //[POST](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/http/latest/http/clients/Client#post) remote function
-        //with a body parts as the payload.
-        mime:Entity[] bodyParts = [part1, part2];
-        response = clientEP->post("/echo", bodyParts);
-        handleResponse(response);
+        //Get a byte stream to a given file.
+        var bStream = io:fileReadBlocksAsStream("./files/logo.png");
+
+        if (bStream is stream<byte[], io:Error>) {
+            //Make a POST request with a byte stream as the payload. Since the file path is static
+            //`<@untainted>` is used to denote that the byte stream is trusted.
```

Comment:
```suggestion
            // Make a POST request with a byte stream as the payload. Since the file path is static
            // `<@untainted>` is used to denote that the byte stream is trusted.
```

Shall we fix everywhere?